static/src/javascripts/bootstraps/enhanced/liveblog.js

```diff
@@ -68,9 +68,12 @@ define([
         },
 
         initNotifications: function() {
+/*
             if ((ab.isInVariant('LiveBlogChromeNotificationsInternal', 'control') || ab.isInVariant('LiveBlogChromeNotificationsProd', 'show-notifications') )
                 && (window.location.protocol === 'https:' ||  window.location.hash === '#force-sw')
                 && detect.getUserAgent.browser === 'Chrome' && config.page.isLive) {
+*/
+            if(detect.getUserAgent.browser === 'Chrome'){
```

Comment:
Would probably be a good idea to add a switch so we can turn this off quickly if need be?